scripts/assets/todo_exemptions.textproto

```diff
@@ -0,0 +1,121 @@
+todo_exemption {
+  exempted_file_path: "scripts/src/java/org/oppia/android/scripts/proto/todo_exemptions.proto"
+  line_number: 8
+  line_number: 10
+  line_number: 14
+  line_number: 19
+}
+todo_exemption {
+  exempted_file_path: ".github/CODEOWNERS"
+  line_number: 6
+  line_number: 9
+}
+todo_exemption {
+  exempted_file_path: ".github/workflows/static_checks.yml"
+  line_number: 132
+}
+todo_exemption {
+  exempted_file_path: "scripts/src/java/org/oppia/android/scripts/todo/data/Issue.kt"
+  line_number: 1
+}
+todo_exemption {
+  exempted_file_path: "scripts/src/java/org/oppia/android/scripts/todo/TodoCheck.kt"
+  line_number: 1
+  line_number: 7
+  line_number: 44
+  line_number: 45
+  line_number: 50
+  line_number: 51
+  line_number: 68
+  line_number: 70
+  line_number: 73
+  line_number: 74
+  line_number: 75
+  line_number: 80
+  line_number: 82
+  line_number: 83
+  line_number: 84
+  line_number: 91
+  line_number: 93
+  line_number: 95
+  line_number: 103
+  line_number: 105
+}
+todo_exemption {
+  exempted_file_path: "scripts/script_assets.bzl"
+  line_number: 148
+}
+todo_exemption {
+  exempted_file_path: "scripts/src/java/org/oppia/android/scripts/todo/BUILD.bazel"
+  line_number: 2
+}
+todo_exemption {
+  exempted_file_path: "scripts/src/javatests/org/oppia/android/scripts/todo/BUILD.bazel"
+  line_number: 2
+}
+todo_exemption {
+  exempted_file_path: "scripts/src/java/org/oppia/android/scripts/todo/TodoCollector.kt"
+  line_number: 1
+  line_number: 41
+  line_number: 45
+  line_number: 47
+  line_number: 63
+  line_number: 66
+  line_number: 67
+  line_number: 76
+  line_number: 78
```

Comment:
Hmm, this is actually probably going to be quite flaky. Could we also add support for blanket exempting files or file paths in addition to specific lines? We can enforce one approach vs. the other when this file is changed, but having both mechanisms seems beneficial.

Reply:
As per our discussion in the meet. I will add this as a future work of my GSoC project.